frontend/package.json

```diff
@@ -10,14 +10,11 @@
   "dependencies": {
     "@material-ui/core": "^4.4.1",
     "@material-ui/icons": "^4.4.1",
-    "@sentry/browser": "^5.6.3",
     "axios": "^0.19.0",
-    "bootstrap": "^4.3.1",
+    "bootstrap": "^4.4.1",
```

Comment:
Is It Necessary to maintain bootstrap? I don't know Where are we continue using :)

Reply:
It's used on the login screen. I will 🔪 on a login screen refactor.